index.js

```diff
@@ -34,32 +34,30 @@ server.on('connect', () => {
     initiateConnection()
 });
 
+// Stop trying to reconnect after 3 attempts
+server.on('reconnect_attempt', connection_attempts => {
+    if (connection_attempts > 3) {
```

Comment:
we generally use camelCase for variables
```suggestion
server.on('reconnect_attempt', connectionAttempts => {
    if (connectionAttempts > 3) {
```